pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/io/buffer/Bytes.java

```diff
@@ -168,6 +169,21 @@ public static Bytes fromHex(@NonNull final String string) {
     // ================================================================================================================
     // Object Methods
 
+    /**
+     * A method to get the data of the Bytes on package level. Used for the fast BufferedData wrapper
+     * around the {@link Bytes} object.
+     * @return the byte[] associated with this {link Bytes} object.
+     */
+    byte[] getBytes() {
+        return buffer;
+    }
+
+    /** Gets the start of this {@link Bytes} objec.
```

Comment:
code formatting and typo

```
/** Gets the start of this {@link Bytes} objec.
```
should be
```
/** 
 * Gets the start of this {@link Bytes} object.
```

running spotless apply should clean up the formatting

Reply:
Fixed.